frameworks/elastic/docs/elastic-x-pack.md

```diff
@@ -12,6 +12,9 @@ enterprise: 'no'
 
 You can toggle this setting at any time. This gives you the option of launching an Elastic cluster without X-Pack and then later enabling it. Or, you can run a cluster with X-Pack enabled to try out the commercial features and, if at the end of the 30-day trial period you don't wish to purchase a license, you can disable it without losing access to your data.
 
+Note that you must set the update strategy to `parallel` when you toggle X-Pack in order to force a full cluster restart.
```

Comment:
IMO this should go first in this section. WDYT, @sascala?

Reply:
agree. it seems cruel to put it later.